internal/proto/controller/api/services/v1/auth_method_service.proto

```diff
@@ -194,7 +194,7 @@ message LdapLoginAttributes {
 
 message AuthenticateRequest {
   // The ID of the Auth Method in the system that should be used for authentication.
-  string auth_method_id = 1 [json_name = "auth_method_id"]; // @gotags: `class:"public"`
+  string auth_method_id = 1 [json_name = "auth_method_id"]; // @gotags: `class:"public" eventstream:"observation"`
```

Comment:
@jimlambrt I was unsure if we should include these `AuthenticateRequest` & `AuthenticateResponse` to this PR. What do you think?

Reply:
I think, we could be okay as long as we're careful about which fields are included.